### PR TITLE
fix: [FFM-4382]: Fix some of the FF Stage Overview styling issues

### PR DESCRIPTION
**Pull request by knagurski**
This PR updates the styling of the FF Stage Overview panel to better match other Stage Overview panels and fixes an issue whereby the _Continue_ button would float over the top of the form on smaller viewport sizes.

|Before|After|
|------|------|
|Before|After|

### CODE REVIEW

**Comment by diptiman-adak** [comment]
Quality gate Quality gate: **_Passed_** 
 


            
 No issues Everything looks good
 [Details](https://app.embold.io/organization/gh/harness/repositories/e864badd0b95ab81c060b15b85f15701/pull-request/11393)

**Review by laura-mannus — DISMISSED** [review]

**Review by crystal-fong — APPROVED** [review]

File: src/modules/75-cf/pages/pipeline-studio/views/FeatureStageSetupShell/FeatureStageSetupShell.module.scss [hunk 1]
```diff
@@ -4,6 +4,7 @@
  * that can be found in the licenses directory at the root of this repository, also available at
  * https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt.
  */
+/* stylelint-disable color-no-hex */
```
Comment (knagurski) [inline src/modules/75-cf/pages/pipeline-studio/views/FeatureStageSetupShell/FeatureStageSetupShell.module.scss:7]: I was gonna fix all the hex codes in this file, but I tracked their origin and the same CSS rules are used in a bunch of places; I figured that if that needs to change in the future, someone will search and replace and end up missing these ones. So opted to just tell Stylelint to chill

File: src/modules/75-cf/pages/pipeline-studio/views/StageOverview/__tests__/FeatureStageSpecifications.test.tsx [hunk 2]
```diff
@@ -85,7 +85,7 @@ const getPipelineContext = (): PipelineContextInterface => ({
 })
 
 describe('StepWidget tests', () => {
-  test(`renders DeployStageSpecifications without crashing `, () => {
+  test('renders StageOverview', () => {
```
Comment (crystal-fong) [inline src/modules/75-cf/pages/pipeline-studio/views/StageOverview/__tests__/FeatureStageSpecifications.test.tsx:88]:
```suggestion
  test('it renders StageOverview', () => {
```

**Review by crystal-fong — DISMISSED** [review]

**Review by laura-mannus — APPROVED** [review]

File: src/modules/75-cf/pages/pipeline-studio/views/StageOverview/__tests__/FeatureStageSpecifications.test.tsx (hunk 2 above)
Comment (knagurski) [reply src/modules/75-cf/pages/pipeline-studio/views/StageOverview/__tests__/FeatureStageSpecifications.test.tsx:88]: Damn... throwing my own nitpicks back at me. Touché

Comment (crystal-fong) [reply src/modules/75-cf/pages/pipeline-studio/views/StageOverview/__tests__/FeatureStageSpecifications.test.tsx:88]: just learnin' from the best

Comment (knagurski) [reply src/modules/75-cf/pages/pipeline-studio/views/StageOverview/__tests__/FeatureStageSpecifications.test.tsx:88]: The student has become the master 🙇🏼

Comment (knagurski) [reply src/modules/75-cf/pages/pipeline-studio/views/StageOverview/__tests__/FeatureStageSpecifications.test.tsx]: Fixed now 👍🏻

Comment (crystal-fong) [reply src/modules/75-cf/pages/pipeline-studio/views/StageOverview/__tests__/FeatureStageSpecifications.test.tsx]: oh you actually updated it even though I approved it anyway

**Review by crystal-fong — APPROVED** [review]

File: src/modules/75-cf/pages/pipeline-studio/views/StageOverview/__tests__/FeatureStageSpecifications.test.tsx (hunk 2 above)
Comment (knagurski) [reply src/modules/75-cf/pages/pipeline-studio/views/StageOverview/__tests__/FeatureStageSpecifications.test.tsx]: Of course... it was a valid nitpick (a.k.a. if I didn't I could never complain about it in your code again)

**Review by laura-mannus — APPROVED** [review]

**Pull request merged**